invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/components/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { Breadcrumb } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+export default class Breadcrumbs extends Component {
+  constructor(props) {
+    super(props);
+    this.elements = props.elements;
+  }
+
+  _renderBreadcrumbElements = () => {
+    const lastElemIdx = this.elements.length - 1;
+    return this.elements.map((element, i) => (
```

Comment:
This iteration probably is going to need a `key` or react will complain about it.